.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: Xuantie RISC-V CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main, master]
```

Comment:
```suggestion
    branches: [main]
```

Reply:
There's only `main` branch in this project.

Reply:
Thank you for pointing that out!